### ISSUE TITLE
Setting of alpha cutoff

### ISSUE THREAD

**Issue by DavideBrex** [dfernandezperez/RNAseq-Snakemake#1]
Hi Daniel!
This pipeline is great, thank you!
I spotted a possible small mistake in the R script _get_DESeq2_contrasts.R_ : [at this line](https://github.com/dfernandezperez/RNAseq-Snakemake/blob/a885d7aef14087a265aababc9edb4a8c56e1e8b0/workflow/scripts/get_DESeq2_contrasts.R#L30)
`res <- results(dds, name = contrast, filterFun = ihw, alpha = 0.05)
`
I think **alpha** should be set equal to _snakemake@params[["pval"]]_, correct me if I am wrong please.
Thanks again,
Davide

**Comment by dfernandezperez** [comment]
Hey Davide!

Thanks a lot for taking the time to look at the code. Actually there is a little bit of story, but technically is a mistake, and the reason why it happened is very stupid hahaha. The problem is that to explain this to a wet scientist is not so easy... In the sense that some (it already happened to me) won't understand/like/trust the fact that by changing the p-value filter the actual p-values will be a little bit different. So I decided to set IHW alpha to 0.05, and although I have the option to set the p-value in filter_deg, I never change it (so it is always 0.05 as in IHW). 

Then besides that, the other reason is that I am a little bit lazy. By adding the option of setting IHW alpha I had to re-organize the folder and file structures of most of the following downstream rules. I should have never coded the option to choose the pvalue of the contrasts, but since I wanted to fix this some day I left it as a reminder. 

Now that you say it I think it may be time to fix it, thanks a lot!
